src/shared/lib/geography/country-levels.js

```diff
@@ -38,6 +38,16 @@ const getLevelData = async level => {
 };
 
 export const getLocationData = async id => {
+  // Split and sum populations for aggregated features (i.e. tricounties)
+  if (id.indexOf('+') !== -1) {
+    const parts = id.split('+');
+    let data = await Promise.all(parts.map(getLocationData));
+    data = data.reduce((a, d) => {
+      a.population += d.population;
+      return a;
```

Comment:
We could also do something with the name here, right? Like joining with &, like `Abc & Def`.

Reply:
Agreed, we should. Right now, it's done manually inside of the scraper.